package.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "triton-prometheus",
     "description": "Prometheus for TritonDC",
-    "version": "1.1.2",
-    "author": "Joyent (joyent.com)",
+    "version": "1.2.0",
```

Comment:
Does anything pulling this in need to update their version in their package.json ?